File: F1Levier/app/src/main/java/com/djira/f1levier/entity/Course.java
package com.djira.f1levier.entity;

import androidx.room.ColumnInfo;
import androidx.room.PrimaryKey;

public class Course {

    // create id column
    @PrimaryKey(autoGenerate = true)
    private int ID;

    // Create text column
    @ColumnInfo
    private String c1;

    // Create text column
    @ColumnInfo
    private String c2;

    // Create text column
    @ColumnInfo
    private String c3;

    // Create text column
    @ColumnInfo
    private String c4;


    // Getter and setter


    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }



}

File: TP4/CourseCODEP25/app/src/main/java/com/utbm/nouassi/manou/coursecodep25/database/entity/Course.kt
package com.utbm.nouassi.manou.coursecodep25.database.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Course(
    @PrimaryKey(autoGenerate = true)
    var id: Int,

    @ColumnInfo(name = "titre")
    val titre: String,

    @ColumnInfo(name = "date")
    val date: String,

    @ColumnInfo(name = "duree")
    var duree: Int
)